Sources/XiEditor/XiWindowController.swift

```diff
@@ -16,10 +16,18 @@ import AppKit.NSWindowController
 
 final class XiWindowController: NSWindowController {
     override func windowTitle(forDocumentDisplayName displayName: String) -> String {
+        var editedIndicator = ""
+        
+        // If there is more than one tab AND the document in question has unsaved changes then show an edited indicator.
+        // When there is only one tab the edited indicator is already built into the macOS close (red circle) button.
+        if NSDocumentController.shared.documents.count > 1, document?.hasUnautosavedChanges ?? false {
+            editedIndicator = "• "
```

Comment:
Having the space be part of the indicator seems somewhat hacky to me. Personally I'd make the sapce part of the format string, or drop it entirely. Not going to block this PR on that though, especially considering the recurring thought of ditching the whole NSDocument infra...